docs/content-development/generating.md

```diff
@@ -349,10 +349,10 @@ Now your Terraform manifests are ready to use! See how can you [test it](#testin
 To test generated manifests you need to:
 
 1. Clone repository with [Hub Manifests](https://github.com/capactio/hub-manifests).
-1. Copy content of the `generated` directory into `manifests` directory.
-1. [Populate the manifests to Public Hub](../example/public-hub-content.mdx#populate-the-manifests-into-hub).
-1. (For AWS only) Create a [Type Instance](../example/typeinstances.md#aws-credentials) with credentials.
-1. Prepare input file `input.yaml`
+2. Copy content of the `generated` directory into `manifests` directory.
```

Comment:
You don't need to enumerate the content with 1,2,3 etc. You can read about it more [here](https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list).

Reply:
Changed to the previous version